### ISSUE TITLE
uhdm-yosys is missing `memory_libmap` command

### ISSUE THREAD

**Issue by xobs** [hdl/conda-eda#321]
The UHDM version of Yosys is missing the `memory_libmap` command:

```
(base) user@Ondo:~$ yosys -q -p memory_libmap
(base) user@Ondo:~$ uhdm-yosys -q -p memory_libmap
ERROR: No such command: memory_libmap (type 'help' for a command overview)
(base) user@Ondo:~$
```

Unfortunately, because UHDM is bundled as its own version of Yosys, this makes it impossible to do memory inference while also using SystemVerilog.

**Comment by xobs** [comment]
Actually, it's possible to copy the plugin from `uhdm-yosys` to `yosys`, in which case it works.

What's the rationale of distributing a second copy of Yosys, as opposed to including the plugin with the standard Yosys distribution?

```sh
$ cp -a "$CONDA_PREFIX/share/uhdm-yosys/plugins" "$CONDA_PREFIX/share/yosys/"
```

**Comment by mithro** [comment]
I believe that Yosys previously needed patches to work with UHDM. I _believe_, that should no longer be the case?

FYI - @proppy / @kgugala / @mgielda

**Comment by mithro** [comment]
FYI - @hzeller / @PiotrZierhoffer

**Comment by xobs** [comment]
Then I think the uhdm plugin for Yosys should be updated so that it installs the `.so` files into `share/yosys/plugins`, and remove the ned for it to bundle its own version of Yosys.

**Comment by tgorochowik** [comment]
Agreed, let's remove the obsolete yosys-uhdm build, everything should work with stock yosys and I don't think there are any users of the obsolete fork at this point
